lib/utilities/pipeline.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var Promise = require('rsvp').Promise;
+var _       = require('lodash-node');
+
+var chalk = require('chalk');
+var blue  = chalk.blue;
+var red   = chalk.red;
+var grey  = chalk.grey;
+
+function Pipeline(hookNames, options) {
```

Comment:
Any reason why we are not using core-object here? The rest of ember-cli and ember-cli-deploy is using core-object when doing `classy` stuff